ksqldb-functional-tests/src/test/java/io/confluent/ksql/test/planned/PlannedTestGeneratorTest.java

```diff
@@ -36,7 +36,7 @@
    * or changed query plans.
    */
   @Test
-  @Ignore
+  //@Ignore
```

Comment:
Should this be uncommented?

Reply:
Just as well I added a meta test that fails if its not ;)